packages/config/config/devices/0x0234/templates/logic_group_template.json

```diff
@@ -0,0 +1,474 @@
+			}
+		]
+	},
+	"dimmer_min": {
+		"$import": "../../templates/master_template.json#base_0-99_nounit",
+		"label": "Dimmer minimum level",
+		"unit": "%",
+		"defaultValue": 0
+	},
+	"dimmer_max": {
+		"$import": "../../templates/master_template.json#base_0-99_nounit",
+		"label": "Dimmer maximum level",
+		"unit": "%",
+		"defaultValue": 99
+	},
+	"led_indicator": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"valueSize": 4,
+		"unsigned": true,
+		"allowManualEntry": true
+	},
+	"pushbutton_led_indicator": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "Pushbutton LED indicator",
+		"maxValue": 7,
+		"defaultValue": 7,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Internal LED disabled, only external commands control the indication",
+				"value": 0
+			},
+			{
+				"label": "LED follows switch",
+				"value": 1
+			},
+			{
+				"label": "LED follows switch, inverted",
+				"value": 2
+			},
+			{
+				"label": "Same as 1",
+				"value": 3
+			},
+			{
+				"label": "Same as 2",
+				"value": 4
+			},
+			{
+				"label": "Follow internal dimmer",
+				"value": 5
+			},
+			{
+				"label": "Follow internal dimmer inverted",
+				"value": 6
+			},
+			{
+				"label": "On for 5 sec",
+				"value": 7
+			}
+		]
+	},
+	"color_commands_pushbutton": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"maxValue": 2,
+		"defaultValue": 1,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Direct control",
+				"value": 0
+			},
+			{
+				"label": "OFF state",
+				"value": 1
+			},
+			{
+				"label": "ON state",
+				"value": 2
+			}
+		]
+	},
+	"binary_switch_set": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"maxValue": 2,
+		"defaultValue": 0,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "LED",
+				"value": 0
+			},
+			{
+				"label": "Switch and LED",
+				"value": 1
+			},
+			{
+				"label": "Pushbutton activated",
+				"value": 2
+			}
+		]
+	},
+	"multilevel_switch_set": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"valueSize": 4,
+		"maxValue": 4294967295,
+		"defaultValue": 33488896,
+		"allowManualEntry": true,
+		"options": [
+			{
+				"label": "Byte 4: not used – must be set to 0",
+				"value": 0
+			},
+			{
+				"label": "Byte 1: disabled – a single activation of the pushbutton will not send commands to devices in association group 4",
+				"value": 0
+			},
+			{
+				"label": "Byte 1: enabled – a single activation will send commands to devices in association group 4. devices will receive commands with the values set in byte 2 and 3 (default)",
+				"value": 1
+			},
+			{
+				"label": "Byte 2: when single pressing the pushbutton for on, a multilevel switch set with this value will be send to devices in association group 4. (default)",
+				"value": 255
+			}
```

Comment:
This smells like partial parameters, but these options make no sense at all.

Reply:
should be resolved in this update that is inc.